src/sagas/wallet.js

```diff
@@ -143,7 +143,7 @@ export function* startWallet(action) {
     }
 
     wallet = new HathorWalletServiceWallet({
-      requestPassword: showPinScreenForResult,
+      requestPassword: () => showPinScreenForResult(dispatch),
```

Comment:
Apparently this was a bug and the app would never refresh its token (after 30 min of usage)